pureport/credentials/endpoint/provider.go

```diff
@@ -81,12 +84,14 @@ func (p *Provider) Retrieve() (credentials.Value, error) {
 	// Create the HTTP Request
 	resp, err := p.Client.Post(fmt.Sprintf("%s/login", p.EndPoint), "application/json", buf)
 	if err != nil {
+		log.Errorf("HTTP Response: %s Error: %s", resp, err)
 		return credentials.Value{ProviderName: providerName}, fmt.Errorf("Error creating credentials login request")
 	}
 	defer resp.Body.Close()
 
 	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Not for this story, but would it be useful to log the response if some sort of verbose logging is enabled?

Reply:
Yes. There is a story open already...https://pureport.atlassian.net/browse/PE-1071